packages/editor/src/store/selectors.js

```diff
@@ -525,6 +525,11 @@ export const isEditedPostAutosaveable = createRegistrySelector( ( select ) => fu
 		return false;
 	}
 
+	// A post is not autosavable when there is a post autosave lock.
+	if ( isPostAutosavingLocked() ) {
```

Comment:
Well, it looks like this addresses my previous comment :)

```suggestion
	if ( isPostAutosavingLocked( state ) ) {
```

I think it should take `state` as an argument to work properly.